### PR TITLE
Use constants for the indentation block type

### PR DESCRIPTION
**Pull request by goetas**

### CODE REVIEW

**Review by greg0ire — DISMISSED** [review]

**Comment by greg0ire** [comment]
:thinking: This should target 1.5.x, right?

File: src/SqlFormatter.php [hunk 1]
```diff
@@ -230,9 +233,9 @@ public function format(string $string, string $indentString = '  '): string
                 // Top level reserved words start a new line and increase the special indent level
                 $increaseSpecialIndent = true;
 
-                // If the last indent type was 'special', decrease the special indent for this round
+                // If the last indent type was self::INDENT_TYPE_SPECIAL, decrease the special indent for this round
```
Comment (mvorisek) [inline src/SqlFormatter.php]:
```suggestion
                // If the last indent type was special, decrease the special indent for this round
```

**Comment by mvorisek** [comment]
Yes, this is definitely a feature.

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @goetas !

**Pull request merged**